implementations/rust/ockam/ockam_api/src/authenticator/storage/authority_token.rs

```diff
@@ -0,0 +1,43 @@
+use crate::authenticator::one_time_code::OneTimeCode;
```

Comment:
Can we call the file `token.rs`, like the data type it contains?

Reply:
I'll rename the file and the struct to EnrollmentToken actually. Token is too broad